firmware/src/main.c

```diff
@@ -17,6 +17,7 @@
 #include "hardware/sync.h"
 #include "hardware/structs/ioqspi.h"
 #include "hardware/structs/sio.h"
+#include "hardware/clocks.h"
```

Comment:
Why?

Reply:
Because it's explictly specified `-Wall -Werror -Wfatal-errors` as compiler options and in line 211 of main.c, there's a warning complains  'set_sys_clock_khz' is implicitly declared without importing this header file.
Well, just out of curiosity, is it because I'm using vscode+pico plugin thus a failure is always guaranteed? What dev tools do you use?

Reply:
![{E126C56C-A9FF-437F-B12D-69EEC57E486F}](https://github.com/user-attachments/assets/228ff113-4304-4a4c-9dc9-539ca1df20b4)
It's in pico/stdlib.h.
So please revert this line and only do the `getchar_*()` fix.